test/test.mli

```diff
@@ -1,11 +1,18 @@
 (*Ideally we should have a unified interface of testing framework, I am still thinking about how to abstract away the difference between JS testing and OUnit.*)
 type ava
-type exe
 
-val test: (exe -> bool) -> unit
+type t
 
-val test_with_msg: string -> (exe -> bool) -> unit
+val test: (t -> unit) -> unit
 
-val truthy: exe -> 'a -> bool 
+val test_with_msg: string -> (t -> unit) -> unit
 
-val falsy: exe -> 'a -> bool
+val truthy: t -> 'a -> unit
```

Comment:
You should really prefer `true` and `false` taking `Js.boolean`s over polymorphic `truthy` and `falsy`. The lastter is a bit too JS.